### DISCUSSION TITLE
Programming request

### DISCUSSION THREAD

**Discussion by mcCrypto4you** [community/community#131296] [Other Feature Feedback, Questions, & Ideas]
Looking for a programmer that hat experience with: Programming language : JavaScript
Library : ReactJS, Google cloud functions, Firebase , NodeJS
Database: Firestore
Design Library: ReactJS Material UI, CSS

**Comment by ghostinhershell** [comment]
Hi @mcCrypto4you, we’re glad you’re here 😀

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
